contracts/OffsetHelper.sol

```diff
@@ -139,11 +139,8 @@ contract OffsetHelper is OffsetHelperStorage {
         // instantiate router
         IUniswapV2Router02 routerSushi = IUniswapV2Router02(sushiRouterAddress);
 
-        // establish path
-        address[] memory path = new address[](3);
-        path[0] = _fromToken;
-        path[1] = eligibleTokenAddresses["USDC"];
-        path[2] = _toToken;
+        // generate path
+        address[] memory path = generatePath(_fromToken, _toToken);
```

Comment:
It probably is the case with Sushi as well. I have not noticed this bug when I first added the Uniswap interfaces so thanks for pointing it out 🙏🏻 

But, `OffsetHelper` actually uses `UniswapV2Router02` even though the interface for `getAmountIn()` (**which OffsetHelper doesn't even use**) isn't within `IUniswapV2Router02.sol`. Also, the method interface itself remains the same on both versions.

So, I guess it's not really a major issue? If you think it's important I can update the Uni interfaces within this PR or do it in a different PR this week. Otherwise, I can put it in my backlog.

What do you think?

Reply:
Not sure I understand, the function signature must be within `IUniswapV2Router02.sol` otherwise, you would not be able to compile. 
And it actually[ is imported in L05.](https://github.com/lazaralex98/example-implementations/blob/c6d2b9fe089b181b3946166bee838fd3df988a85/contracts/interfaces/IUniswapV2Router02.sol#L5)

Reply:
It is indeed imported. But it's literally the same interface regardless of wether it's imported or actually in `IUniswapV2Router02`. Also, yes it would compile as I just tried removing `getAmountIn()` altogether. Either way, I'll just update the interfaces in this PR.